ovos_padatious/training_manager.py

```
@@ -84,10 +84,15 @@ def add(self, name: str, lines: List[str], reload_cache: bool = False, must_trai
                     old_hsh = g.read()
             min_ver = splitext(ovos_padatious.__version__)[0]
             new_hsh = lines_hash([min_ver] + lines)
-            if reload_cache or old_hsh != new_hsh:
+            retrain = reload_cache or old_hsh != new_hsh
+            if not retrain:
+                try:
+                    self.objects.append(self.cls.from_file(name=name, folder=self.cache))
+                except:
```

Ruff (0.8.0)

Do not use bare `except`

(E722)